### ISSUE TITLE
HomeAssistant Assist request?

### ISSUE THREAD

**Issue by Panda260** [avraham/hass_radarr_sonarr_search_by_voice#23]
Hey @avraham 
Would it be possible to update the repo to make it work again? I would be happy if there was something like that!
2023 was the “HomeAssistant year of the voice” and it is becoming more and more developed. Wouldn’t it really be easiest and best to implement it directly with the integrated HomeAssistant Assist? It would be a certain amount of work, but it would be a way to revive the project

**Comment by avraham** [comment]
Hi,
I didn't know about Home Assistant Assist, I haven't updated home assistant in a year, but it looks really interesting!!

I can't promise anything as this is just a hobby on my spare time, but I'll take a look into it.
Do you happen to know if "Home Assistant Assist" works on a supervised installation too?, I had a quick look in the documentation an they mention a Home Assistant OS installation.

**Comment by Panda260** [comment]
If I understand your question correctly...
then yes the Assist is built into Homeassistant and can be used locally on your own hardware for local voice control or optionally via the cloud
I don't really know what you mean by supervised, but Home has a supervised installation as its own OS, which means you can actually install it anywhere on an Intel NUC, Raspberry Pi or in a VM, that's possible. or but there Hardware with HomeAssistant already installed.

You can use the HomeAssistant assistant, for example: run on an ESP32, which serves as a remote microphone/speaker, so to speak, and then sends the data to homeassistant.

**Comment by avraham** [comment]
I just meant the installation method.
[https://www.home-assistant.io/installation/#compare-installation-methods](https://www.home-assistant.io/installation/#compare-installation-methods)

Yes, I guess I'll have no problem with my setup.
Thanks.

**Comment by Panda260** [comment]
So you think you can do it and also have the desire/time to update the repo for the assist
That would be really cool!

**Comment by Panda260** [comment]
@avraham any updates??

**Comment by avraham** [comment]
I looked at it. The best way would be if this functionality is implemented
inside a custom component.

This would be my first custom component in a few years and I want to make
sure I follow the guidelines.

Do you any coding knowledge?, Maybe we could help each other.

I'm slow, but I intend to do it :)

**Comment by Panda260** [comment]
Write to me on discord @panda260

**Comment by colethegamer123** [comment]
Hello Guys! I does not have deep developing knowledge only scripting via AI :D but maybe I could help with testing and solve issues.

I have a radarr and sonarr setup and I have assist in my own language and I have also the new Home Assistant Preview PE device for Assist.

It would be really great to have something like this, biggest invention since fire 
Thank you guys!

**Comment by avraham** [comment]
@colethegamer123 have you looked at this?
[https://github.com/avraham/hass_radarr_actions](https://github.com/avraham/hass_radarr_actions)

**Comment by colethegamer123** [comment]
> [@colethegamer123](https://github.com/colethegamer123) have you looked at this? https://github.com/avraham/hass_radarr_actions

Yes mate, I also spoke with Panda on discord. I totally understand what you did and mostly understand the how too.

It can be a perfect solution now, where you can use multiple solutions like ChatGPT and Yaris in the same time, but Yarvis does not work anymore, I cannot install it because giving error message when I want to add to integrations. I also opened an issue there, but no response anymore as I see.

Your solution is working by text but I cannot use without Yaris, but I think its also possible now without Yaris I just dont understand how to use variables with assist.

If you will have a few minute could you check on it? Only the movie name recognition does not work by voice, everything else is still working.

Thanks for your time, I really appreciate this whole project!
Gery
